### PR TITLE
chore(package): remote .ts files from built package

### PR DESCRIPTION
**Pull request by ajcrites**
Currently the `.ts` files are also being published.

### CODE REVIEW

**Comment by ajcrites** [comment]
I've updated the PR to fix what you brought up.

File: .npmignore [hunk 1]
```diff
@@ -2,3 +2,7 @@
 
 tsconfig.json
 tslint.js
+.circleci
+*.ts
+!*.d.ts
+!schematics/page/files/**/*.ts
```
Comment (dwieeb) [inline .npmignore]: This could be `!schematics/*/files/**/*.ts` to avoid any accidental exclusion of `.ts` files for new schematics.

Comment (ajcrites) [reply .npmignore]: Updated

**Comment by dwieeb** [comment]
Thank you @ajcrites! This is great 👍

**Pull request merged**